curation_concerns.gemspec

```diff
@@ -45,6 +45,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'dry-struct', '~> 0.1'
   spec.add_dependency 'flot-rails', '~> 0.0.7'
   spec.add_dependency 'highcharts-rails'
+  spec.add_dependency 'hydra-pcdm', '~> 0.9.0'
```

Comment:
I've uddated https://github.com/projecthydra/hydra-works/pull/303 to make HW require hydra-pcdm `>= 0.9.0`.  I can update this PR to use that once it's released.